torch/csrc/distributed/c10d/init.cpp

```diff
@@ -284,7 +284,11 @@ They are used in specifying strategies for reduction collectives, e.g.,
 
   auto store =
       py::class_<::c10d::Store, std::shared_ptr<::c10d::Store>, PythonStore>(
-          module, "Store")
+          module, "Store",
+          R"(
+Base class for the 3 store implementations: (:class:`~torch.distributed.TCPStore`,
```

Comment:
Probably just mention this is the base class for all store implementations. There could be other user implementations on top of this apart from these 3.